test/plugin/scenarios/lettuce-scenario/support-version.list

```diff
@@ -16,4 +16,7 @@
 
 5.2.1.RELEASE
 5.1.8.RELEASE
-5.0.5.RELEASE
\ No newline at end of file
+5.0.5.RELEASE
+6.0.0.RELEASE
+6.0.5.RELEASE
```

Comment:
```suggestion
6.0.5.RELEASE
```

We recommend keep one version for minor version, and ignore revision to reduce resource cost.

Reply:
removed